### PR TITLE
Change python/pip references to python3/pip3

### PR DESCRIPTION
**Pull request by hshin-g**
Change python/pip references to python3/pip3 to align with Python 3 migration efforts for v2.16.0.

Align with `fs-future` branch on `forseti-security`.

### CODE REVIEW

**Comment by morgante** [comment]
@hshin-g Is this safe to merge now, or is it pending the release of the next Forseti version?

**Comment by hshin-g** [comment]
@morgante This is pending the release of the next Forseti version.

**Comment by morgante** [comment]
@hshin-g How about we start a release branch and merge this into there?

File: modules/client/templates/scripts/forseti-client/forseti_client_startup_script.sh.tpl [hunk 1]
```diff
@@ -32,13 +32,13 @@ git checkout ${forseti_version}
 sudo apt-get install -y $(cat install/dependencies/apt_packages.txt | grep -v "#" | xargs)
 
 # Forseti dependencies
-pip install --upgrade pip==9.0.3
-pip install -q --upgrade setuptools wheel
-pip install -q --upgrade -r requirements.txt
+pip3 install --upgrade pip==19.1.1
```
Comment (joecheuk) [inline modules/client/templates/scripts/forseti-client/forseti_client_startup_script.sh.tpl]: Can you revert back to 9.0.3 for now? This breaks some of the code as the pip python library changed.

File: modules/server/templates/scripts/forseti-server/forseti_server_startup_script.sh.tpl [hunk 2]
```diff
@@ -43,9 +43,9 @@ git checkout ${forseti_version}
 sudo apt-get install -y $(cat install/dependencies/apt_packages.txt | grep -v "#" | xargs)
 
 # Forseti dependencies
-pip install --upgrade pip==9.0.3
-pip install -q --upgrade setuptools wheel
-pip install -q --upgrade -r requirements.txt
+pip3 install --upgrade pip==19.1.1
```
Comment (joecheuk) [inline modules/server/templates/scripts/forseti-server/forseti_server_startup_script.sh.tpl]: Same for this one

File: modules/client/templates/scripts/forseti-client/forseti_client_startup_script.sh.tpl (hunk 1 above)
Comment (hshin-g) [reply modules/client/templates/scripts/forseti-client/forseti_client_startup_script.sh.tpl]: Changed.

File: modules/server/templates/scripts/forseti-server/forseti_server_startup_script.sh.tpl (hunk 2 above)
Comment (hshin-g) [reply modules/server/templates/scripts/forseti-server/forseti_server_startup_script.sh.tpl]: Changed.

**Pull request merged**